portmapper/proxy.go

```diff
@@ -65,10 +66,23 @@ func (p *proxyCommand) Start() error {
 
 func (p *proxyCommand) Stop() error {
 	if p.cmd.Process != nil {
-		if err := p.cmd.Process.Signal(os.Interrupt); err != nil {
+		if err := p.cmd.Process.Signal(syscall.SIGTERM); err != nil {
```

Comment:
What is the reason to replace SIGINT with SIGTERM?

Reply:
was wondering as well; should we handle _both_ ? (for backward compat)?

Reply:
To be honest, i‘’m  also not sure why send SIGINT  other than SIGTERM.  Proxy  will  deal with both of them . According appearance of problem， SIGINT looks can't got response.   Besides,  semantically speaking，SIGTERM could be suitable.  so , i  make this change.